### ISSUE TITLE
State of this project?

### ISSUE THREAD

**Issue by dominikonysz** [RLE-Foundation/RLeXplore#12]
Hey,

I was just wondering what the state of this project is, since there has been new activity recently. I understand that it had been merged with RLLTE which rather seems to be a replacement for stable-baselines3 than an addition to it's ecosystem.
My interest in particular is to combine MaskablePPO with ICM, which this project would have been ideal for since the other projects are missing one of the building blocks each, unfortunately.

So is there any interest in continuing the development of exploration add-ons for stable-baselines3?

**Comment by yuanmingqi** [comment]
Of course, we will publish a big update recently. You can check this branch [https://github.com/RLE-Foundation/rllte/blob/reward/](https://github.com/RLE-Foundation/rllte/blob/reward/) for the lastest code and there is an example [https://github.com/RLE-Foundation/rllte/blob/reward/experimental/rllte_to_sb3.py](https://github.com/RLE-Foundation/rllte/blob/reward/experimental/rllte_to_sb3.py) for using RLeXplore with SB3.

**Comment by dominikonysz** [comment]
Great, thank you!

Unfortunately I'm having some issues with installing it, but for that see https://github.com/RLE-Foundation/rllte/issues/44.
